### PR TITLE
Improve remark on UDP/TCP for high availability

### PR DESCRIPTION
**Pull request by pascal-hofmann**
A lot of people (including myself) seem to struggle with setting up high availability. I think it makes sense to improve the remarks on UDP/TCP for high availability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,6 +331,8 @@ $ amtool config routes test --config.file=doc/examples/simple.yml --tree --verif
 Alertmanager's high availability is in production use at many companies and is enabled by default.
 
 > Important: Both UDP and TCP are needed in alertmanager 0.15 and higher for the cluster to work.
+>  - If you are using a firewall, make sure to whitelist the clustering port for both protocols.
+>  - If you are using docker, make sure to expose the clustering port for both protocols (i.e. `-p 9094:9094/tcp -p 9094:9094/udp`)
```
Comment (mxinden) [inline README.md:335]:
```suggestion
>  - If you are using docker, make sure to expose the clustering port for both protocols (i.e. `-p 9094:9094/tcp -p 9094:9094/udp`).
```

File: README.md [hunk 2]
```diff
@@ -331,6 +331,8 @@ $ amtool config routes test --config.file=doc/examples/simple.yml --tree --verif
 Alertmanager's high availability is in production use at many companies and is enabled by default.
 
 > Important: Both UDP and TCP are needed in alertmanager 0.15 and higher for the cluster to work.
+>  - If you are using a firewall, make sure to whitelist the clustering port for both protocols.
+>  - If you are using docker, make sure to expose the clustering port for both protocols (i.e. `-p 9094:9094/tcp -p 9094:9094/udp`).
```
Comment (simonpasquier) [inline README.md:335]: "if you are using docker" -> "if you are running in a container"

Comment (brian-brazil) [reply README.md:335]: Containers don't have command line flags.

Comment (simonpasquier) [reply README.md:335]: Hence removing the (i.e. ...) part.

Comment (pascal-hofmann) [reply README.md]: Changed.

**Comment by paulfantom** [comment]
@pascal-hofmann you need to sign off your last 3 commits to pass DCO check. Just run:
```
git filter-branch -f --commit-filter 'git commit-tree -S "$@"' HEAD~3..HEAD
git push --force
```

**Comment by paulfantom** [comment]
@pascal-hofmann you need to sign off your last 3 commits to pass DCO check. Just run:
```
git filter-branch --msg-filter 'cat - && echo && echo "Signed-off-by: Pascal Hofmann <mail+github@pascalhofmann.de>
" "$@";' HEAD~3..HEAD
git push --force
```

**Comment by pascal-hofmann** [comment]
Thanks for the detailed instructions and sorry for the delay. It was a busy week for me… :)

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**